src/core/lib/security/security_connector/ssl/ssl_security_connector.cc

```diff
@@ -204,8 +171,9 @@ class grpc_ssl_channel_security_connector final
                          overridden_target_name_.c_str(), auth_context));
   }
 
+  tsi_ssl_client_handshaker_factory* client_handshaker_factory_ = nullptr;
```

Comment:
Please make this a private member field.

Reply:
Made it private and then added it to the constructor of `grpc_ssl_channel_security_connector` PTAL